addons/point_of_sale/static/src/js/models.js

```diff
@@ -2070,7 +2070,7 @@ exports.Orderline = Backbone.Model.extend({
     },
     _map_tax_fiscal_position: function(tax) {
         var self = this;
-        var current_order = this.pos.get_order();
+        var current_order = this.order || this.pos.get_order();
```

Comment:
@pimodoo May I have your opinion on this? 
I don't really get why when fetching the order with `get_order()` we don't have the fiscal position